src/web/page/templates.rs

```diff
@@ -99,13 +82,28 @@ fn load_rustc_resource_suffix() -> Result<String> {
     failure::bail!("failed to parse the rustc version");
 }
 
-pub(super) fn load_templates() -> TeraResult<Tera> {
+pub(super) fn load_templates() -> Result<Tera> {
     let mut tera = Tera::new("tera-templates/**/*")?;
 
     // Custom functions
-    tera.register_function("global_alert", global_alert);
-    tera.register_function("docsrs_version", docsrs_version);
-    tera.register_function("rustc_resource_suffix", rustc_resource_suffix);
+    ReturnValue::add_function_to(
+        &mut tera,
+        "global_alert",
+        serde_json::to_value(crate::GLOBAL_ALERT)?,
+    );
+    ReturnValue::add_function_to(
+        &mut tera,
+        "docsrs_version",
+        Value::String(crate::BUILD_VERSION.into()),
+    );
+    ReturnValue::add_function_to(
+        &mut tera,
+        "rustc_resource_suffix",
+        Value::String(load_rustc_resource_suffix().unwrap_or_else(|err| {
+            log::error!("Failed to load rustc resource suffix: {:?}", err);
+            String::from("???")
+        })),
+    );
```

Comment:
Maybe add back the comments that were on the struct so the function's meaning is clear(er)